src/main/webapp/login.jsp

```diff
@@ -13,6 +13,9 @@
       <a href="${loginUrl}">
         <button>Log In</button>
         </a>
+      <a href="${loginUrlNewUser}">
```

Comment:
This link doesn't direct to anything (even though in CreateNewUser it's defined as a login link that redirects to create-new-user.jsp, which is a form)